acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Puppetserver 'foreground' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
```

Comment:
Yeah, good catch.  If it's not too much work -- like no more than a few minutes -- I'd probably just make the necessary updates to the `gem.rb` test to reflect this as well.